### DISCUSSION TITLE
Any chance to be able to restore or boot to iOS 5.x without blobs on iPhone 4s?

### DISCUSSION THREAD

**Discussion by mailinglists35** [LukeZGD/Legacy-iOS-Kit#436] [Q&A] [answered]
Model MD242D/A
manufactured week 39 of 2011
kok3shi works on 9.3.6
coolbooter works and boots 6.1.3 and setups Cydia, but only shows iOS 6 as the lowest.

**Comment by LukeZGD** [comment, marked as answer]
Without blobs, your only option is dualbooting (Coolbooter), or downgrading tethered (it is an option shown as "Other (Tethered)" in Legacy iOS Kit).

Tethered downgrades for A5 devices such as the 4S will require an Arduino+USB Host Shield, since tether booting requires pwning the device with checkm8-a5.

https://github.com/LukeZGD/Legacy-iOS-Kit/wiki/Troubleshooting#dfu-advanced-menu-a5x-pwndfu-mode-with-arduino-and-usb-host-shield

**Reply by mailinglists35** [reply]
> Without blobs, your only option is dualbooting (Coolbooter)

ah, this implies using the cli version, correct?

**Reply by LukeZGD** [reply]
Yes

**Reply by mailinglists35** [reply]
hooray, it worked using the cli via ssh
